pcdsdevices/sqr1.py

```diff
@@ -0,0 +1,43 @@
+"""
+Ophyd class for square-one tri-sphere motion system (https://www.sqr-1.com/tsp.html)
+"""
+
+import logging
+
+from ophyd.device import Device
+from ophyd.device import FormattedComponent as FCpt
+from ophyd.pv_positioner import PVPositionerIsClose
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+logger = logging.getLogger(__name__)
+
+
+class SQR1Axis(PVPositionerIsClose):
+    setpoint = FCpt(EpicsSignal, '{self.prefix}:TARGET:{self._axis}', kind='normal')
```

Comment:
(Optional) There's a shorthand for these: "self." can be omitted.
```suggestion
    setpoint = FCpt(EpicsSignal, '{prefix}:TARGET:{_axis}', kind='normal')
```